components/bintray-publish/bin/publish-studio.sh

```diff
@@ -219,4 +220,5 @@ if [[ -z "${BINTRAY_KEY:-}" ]]; then
 fi
 
 _find_system_commands
+# shellcheck disable=2119
```

Comment:
```
In ./components/bintray-publish/bin/publish-studio.sh line 222:
_main
^-- SC2119: Use _main "$@" if function's $1 should mean script's $1.
```
Either `_main` should be changed to `_main "$@"` or fix https://github.com/habitat-sh/habitat/pull/4974/files#r192900450